lib/ecto/query/planner.ex

```diff
@@ -1783,7 +1783,7 @@ defmodule Ecto.Query.Planner do
 
   @all_exprs [with_cte: :with_ctes, distinct: :distinct, select: :select, from: :from, join: :joins,
               where: :wheres, group_by: :group_bys, having: :havings, windows: :windows,
-              combination: :combinations, order_by: :order_bys, limit: :limit, offset: :offset]
+              combination: :combinations, order_by: :order_bys, limit: :limit, offset: :offset, update: :updates]
```

Comment:
Should be `:update`, no? We don't allow multiple update exprs

Reply:
[This test for allowing update](https://github.com/elixir-ecto/ecto_sql/pull/532/commits/f101edf52773ed583650df77bdfe97fd029f0a82#diff-df64a1e98eb69a93a67e388be3fb937b476bc769705a2a682d3e8e1ecdc8a38c) statements inside PostgreSQL CTEs fails unless `update: :updates`; I had tried all other variants for keys and nothing else works. Not sure I understand what you mean by multiple update exprs?

Reply:
oh, its indeed called `:updates` elsewhere, mb